context/app/static/js/shared-styles/alerts/AlertMessages/AlertMessages.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Is this pattern of showing either warning or error messages something we want generalized in our shared components or is it specific to this case? If so could we use a more descriptive component name which captures that these are warning and error messages? Alerts can be used for purposes other than warnings and errors.

Reply:
That's a good point - I think a more specific name would be fitting (maybe`ErrorOrWarningMessages`).